sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/timewindow/recurrence/RecurrenceConstants.java

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.feature.management.implementation.timewindow.recurrence;
+
+public final class RecurrenceConstants {
+    private RecurrenceConstants() {
+    }
+
+    // Day of week
+    public static final String SUNDAY = "sunday";
```

Comment:
Can we switch to use `DayOfWeek` it's an existing enum, we can just use `DayOfWeek.SUNDAY.name()`

Reply:
Remove these constant strings and use `DayOfWeek` instead